jobs/fs-integration.yml

```diff
@@ -69,7 +69,9 @@
 
     builders:
     - shell: !include-raw-escape: scripts/common/get-node.sh
-    - shell: jobs/scripts/common/bootstrap.sh $WORKSPACE/jobs/scripts/fs-integration/fs-integration.sh "ghprbPullId=$ghprbPullId ghprbTargetBranch=$ghprbTargetBranch CENTOS_VERSION=$CENTOS_VERSION FILE_SYSTEM=$FILE_SYSTEM GIT_REPO=$GIT_REPO"
+    - shell: |
+        #!/bin/sh -e
```

Comment:
It's not clear to me the reason for these "embedded scripts" wrapping bootstrap.sh and friends are for. At first I thought it was to easily put a -x back if needed for temporary work but then I realized that it wouldn't apply to the actual bootstrap.sh and other scripts. So I'm a bit puzzled by these lines.

Reply:
`!include-raw-escape` tag dumps the entire script content.

It's little confusing as [bootstrap.sh](https://github.com/samba-in-kubernetes/samba-centosci/blob/main/jobs/scripts/common/bootstrap.sh) is designed to accept the required script(to be executed on reserved node from CentOS pool via [get-node.sh](https://github.com/samba-in-kubernetes/samba-centosci/blob/main/jobs/scripts/common/get-node.sh)) as an argument by copying it to the remote node. Whereas _bootstrap.sh_ itself is executed inside a Pod created when a job is triggered.

In addition to that script/commands defined for `shell:` is launched as `#!/bin/sh -ex` by default. But we are not interested in displaying whatever is passed onto _bootstrap.sh_ and thus we explicitly avoid `-x`.  On the other side `-e` is already set independently within _bootstrap.sh_.

Does that make sense?